dir.props

```diff
@@ -2,6 +2,12 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
 
+  <!-- We shipped assembly file version 4.6.x up until the end of rc3.  Version assembly as 
+        4.6.x to ensure compatability in Visual Studio for an in-place update. -->
+  <PropertyGroup>
+    <MajorVersion>4</MajorVersion>
+    <MinorVersion>6</MinorVersion>
```

Comment:
I tracked down the specific build where the versioning changed so that I could confirm anything we produced now was >= what was shipped, it was during the RC3 timeframe. However, yes, without additional data points, I didn't know what other version number to go with.